### PR TITLE
Added options to add changelog from a specific commit and some report format additions

### PR DESCRIPTION
**Pull request by VictorDM**
Added includeCommitsAfterCommit option.
Added showCommiter option to hide info about the commiter.
Added markdownChangeLogAppend to append info in markdown file.

### CODE REVIEW

File: src/main/java/com/github/danielflower/mavenplugins/gitlog/GenerateMojo.java [hunk 1]
```diff
@@ -64,7 +64,7 @@
 	 * If true, then an Asciidoc changelog will be generated.
 	 */
 	@Parameter(defaultValue = "false")
-	private boolean generatAsciidocChangeLog;
+	private boolean generateAsciidocChangeLog;
```
Comment (danielflower) [inline src/main/java/com/github/danielflower/mavenplugins/gitlog/GenerateMojo.java]: This is a breaking change... so the misspelled property needs to remain there, but should be marked as deprecated, and should be used if it is true. Or you could revert this change.

**Comment by VictorDM** [comment]
Fixed breaking change. Added correct properties and marked the misspelled ones as deprecated.

**Comment by danielflower** [comment]
Thanks. Released in `1.13.12`
